rtf-factory/src/com/axonivy/utils/rtffactory/RtfExpander.java

```diff
@@ -55,7 +55,7 @@ public static String expand(String text, Object in) throws IvyScriptParserExcept
 		}
 		IEmbeddedScriptExpander macroExpander = EmbeddedScriptExpanderFactory.createExpander();
 		IIvyScriptContext context = request.getContext(null);
-		IIvyScriptEngine engine = request.getProject().getIvyScriptEngine();
+		IIvyScriptEngine engine = request.getScriptEngine();
```

Comment:
Hi @ivy-bb , 
I am converting this product from Ivy 10 to Ivy 12.
In Ivy 12, `request.getProject().getIvyScriptEngine()` no longer exists. I found that we can use `getScriptEngine` directly, so that's why I changed the code here.

Do you agree with this? I tested it and it worked normally.